### ISSUE TITLE
关于batch_size的设置

### ISSUE THREAD

**Issue by Smallwhite999** [chenyuntc/Simplified-FasterRCNN#208]
代码中的batch_size为什么只能设置为1啊？如何去修改为其它的值呢？

**Comment by xiangweifeng** [comment]
看显存大小吧

**Comment by LGsmile** [comment]
因为里面很多代码都是针对一个batch写的，所以想用大于1的batch_size训练要自己修改代码

**Comment by fnsflm** [comment]
batch_size是无法直接修改的，因为在data/dataset.py里定义的训练集Dataset将返回元素定义为`img, bbox, label, scale`而bbox的个数不同图片是不同的，维度不同的bbox无法合并成一个Tensor，这会引起torch.stack报错。需要去重新定义Dataset和读入图片及标签的逻辑才能修改batch_size。
